Doc/cmus.txt

```diff
@@ -518,6 +518,7 @@ push <text>
 		bind common w push filter artist=
 
 	Text can contain spaces and even trailing spaces will be honored.
+	Text can also be left blank.
```

Comment:
You'll want to change `push <text>` to `push [text]` too.

Reply:
There's probably no need for that sentence then.

Reply:
In that case, you'll also want to update the description to "... command line pre-set to text **if provided**".